code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -467,6 +467,15 @@ var/const/NO_EMAG_ACT = -50
 	access = list(access_legion, access_maint_tunnels, access_external_airlocks, access_security, access_engine, access_engine_equip, access_medical, access_research, access_atmospherics, access_medical_equip)
 	..()
 
+/obj/item/card/id/distress/iac
+	name = "\improper Interstellar Aid Corps ID"
```

Comment:
Would be nice to get some special ID cards / description text for it

Reply:
probably